### PR TITLE
Dockerfile fixes

### PR DESCRIPTION
**Pull request by eivantsov**
### What does this PR do?

1. Fixes permissions for user home directory introduced by PR #131 sshd is not less picky about key permissions (important for Codenvy)
2. Fix php5.6 image build
3. Add Rlang stack
4. Update golang version to 1.8.3

### CODE REVIEW

**Comment by benoitf** [comment]
@l0rd

File: recipes/stack-base/ubuntu/Dockerfile [hunk 1]
```diff
@@ -28,7 +28,7 @@ RUN apt-get update && \
     mkdir /var/run/sshd && \
     sed 's@session\s*required\s*pam_loginuid.so@session optional pam_loginuid.so@g' -i /etc/pam.d/sshd && \
     echo "%sudo ALL=(ALL) NOPASSWD: ALL" >> /etc/sudoers && \
-    useradd -u 1000 -G users,sudo -d /home/user --shell /bin/bash -m user && \
+    useradd -u 1000 -G users,sudo -g 0 -d /home/user --shell /bin/bash -m user && \
```
Comment (l0rd) [inline recipes/stack-base/ubuntu/Dockerfile]: If this modification (`-g 0`) is necessary it should be necessary for debian and centos too.

Comment (eivantsov) [reply recipes/stack-base/ubuntu/Dockerfile]: Ah, that was left after some experiments from yesterday. Removed it

**Comment by eivantsov** [comment]
@l0rd is it ok to merge now?

**Pull request merged**